pkg/ipcache/kvstore.go

```diff
@@ -271,7 +272,7 @@ restart:
 					listener.OnIPIdentityCacheGC()
 				}
 				IPIdentityCache.Unlock()
-				close(iw.synced)
+				iw.closeSynced()
```

Comment:
This is part of a "restart" block. Does this mean that if kvstore resynchronization must happen, we will start leaking these channels?

Reply:
I guess not because it's part of the current IPIdentityWatcher `iw` which only really signals synchronization once to the main daemon code. Never mind :+1: